cobra/evaluation/pigs_tables.py

```diff
@@ -213,7 +211,7 @@ def plot_incidence(pig_tables: pd.DataFrame,
 
         # Set labels & ticks
         ax2.set_xlabel('{} bins' ''.format(variable), fontsize=16)
-        ax2.xaxis.set_tick_params(rotation=45, labelsize=14)
+        ax2.xaxis.set_tick_params(rotation=90, labelsize=14)
```

Comment:
If modifying X axis label stuff due to my above comment, this will also be impacted.